src/main/interface/settings.c

```diff
@@ -682,6 +682,7 @@ const clivalue_t valueTable[] = {
     { "use_cbat_alerts",            VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_BATTERY_CONFIG, offsetof(batteryConfig_t, useConsumptionAlerts) },
     { "cbat_alert_percent",         VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 100 }, PG_BATTERY_CONFIG, offsetof(batteryConfig_t, consumptionWarningPercentage) },
     { "vbat_cutoff_percent",           VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 100 }, PG_BATTERY_CONFIG, offsetof(batteryConfig_t, lvcPercentage) },
+    { "force_battery_cell_count",   VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 80 }, PG_BATTERY_CONFIG, offsetof(batteryConfig_t, forceBatteryCellCount) },
```

Comment:
`80` is probably unreasonably high, it may mislead users (well, Tesla uses 99 cells, but I doubt someone is installing BF there ;-) )
12 (maybe 24 for 2 12 cell packs in series) may be more reasonable ...

Reply:
I thought 80 stood for 8, as 41 stood for 4.1 on volrage.

Reply:
voltage in 0.1V increments (0-25.6V)
cell count is direct. And 8 is little bit too small - it is used in existing code because autodetection is useless for more cells

Reply:
okay, 24 it is.